internal/converter/types.go

```diff
@@ -261,6 +261,9 @@ func (c *Converter) convertField(curPkg *ProtoPackage, desc *descriptor.FieldDes
 			} else {
 				jsonSchemaType.Properties = recursedJSONSchemaType.Properties
 				jsonSchemaType.Ref = recursedJSONSchemaType.Ref
+				if c.AllFieldsRequired || desc.GetLabel() == descriptor.FieldDescriptorProto_LABEL_REQUIRED {
```

Comment:
The repeated field in pb 3 should not be append to Required field even though c.AllFieldsRequired is true.